Rakefile

```diff
@@ -36,3 +36,36 @@ if File.exist?(build_defs_file)
     end
   end
 end
+
+desc "verify that commit messages match CONTRIBUTING.md requirements"
+task(:commits) do
+  commits = ENV['TRAVIS_COMMIT_RANGE'] || 'master..HEAD'
```

Comment:
It shouldn't ever happen when running from Jenkins. That's basically a fallback default; I guess I could see changing it to `stable..HEAD` but also, what about not running it at all if `TRAVIS_COMMIT_RANGE` is undefined?